tests/expr_and_series/transform_test.py

```diff
@@ -0,0 +1,16 @@
+from typing import Any
+
+import pytest
+
+import narwhals.stable.v1 as nw
+from tests.utils import compare_dicts
+
+
+def test_transform(constructor: Any, request: Any) -> None:
```

Comment:
I think this test name was probably originally a typo. the functionality seems covered by the `is_between` and `is_in` tests

if you remove this one, and coverage is unaffected, maybe we can just remove it?

Reply:
Everything passed without it. Is there something specific I should check to see whether the coverage changed?

Reply:
you can use `pytest tests --cov=narwhals` to see coverage